WebChannelAPI.md

```diff
@@ -0,0 +1,474 @@
+# Meld Studio's WebChannel API Documentation
+
+## Overview
+
+Meld Studio's WebChannel API enables JavaScript clients to interact with various components of Meld Studio. It allows for the management of `scenes`, `audio tracks`, `effects`, and other `session` states, providing an RPC-like link between the web clients and the application's backend.
+
+### Qt's WebChannel Protocol
+
+Meld Studio's WebSocket API is managed by a `QWebChannel` protocol server. This is used to provide a Qt-like API in Javascript. 
+
+A usable Javascript `QWebChannel` client protocol implementation can be found here: https://cdn.jsdelivr.net/npm/qwebchannel@6.2.0/qwebchannel.min.js
```

Comment:
This is packaged by a third party, we should consider switching to ship our own version to this CDN in the future.

Reply:
You begin referring to `signals` and `slots` below, but it would be good to introduce those concepts here, perhaps with a link to https://doc.qt.io/qt-6/signalsandslots.html for further reading.

Reply:
Fixed in 7d676dc136ad0ef8390ce78eeca8b749bd479623